bin/prefix-vendor-namespace.php

```diff
@@ -141,3 +147,50 @@ function replace_in_json_file( string $file, string $namespace, string $new_name
 		)
 	);
 }
+
```

Comment:
Minor: For our own benefit, we should include a docblock describing what this function accomplishes.